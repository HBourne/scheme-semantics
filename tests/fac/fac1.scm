//factorial with do loop
[define fac
    [lambda [x]
        [if [eqv? x 0] 
            1
            [* x [fac [- x 1]]]]]]
[define input 8]
[fac 8]
eof
