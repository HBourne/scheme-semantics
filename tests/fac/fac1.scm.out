V ==K 40320
